### DISCUSSION TITLE
Several questions: paths, duplicates etc

### DISCUSSION THREAD

**Discussion by adminmat** [immich-app/immich#6554] [Q&A]
I set up Immich on unRAID with the imagegenius' template. There isn't much support in the unRAID forums for Immich so I hope I can ask here. I've uploaded my google Takeout images and everything is working in Immich. Can you help with the following:

I'm not sure I have the paths configured correctly as all my photo files remain in the **/import** (mapped to: /mnt/user/Photos/) container path. I assumed after scanning the files would be moved to **/photos** (mapped to: /mnt/user/immichphotos/).  /import only contains thumbnails but has a directory called **libraby** which is empty. Is this the intended behavior?

here is a screenshot of my template. Sorry, I do not know how to find my docker-compose.yml file..

<img width="752" alt="Screenshot 2024-01-21 123833" src="https://github.com/immich-app/immich/assets/64502231/1f0822a5-8f62-4eb5-b21d-02e09298df35">
<img width="763" alt="Screenshot 2024-01-21 123816" src="https://github.com/immich-app/immich/assets/64502231/e278cec0-a300-4499-a4f6-87c7c5a8d19c">

**Comment by martabal** [comment]
`/import` is an optional path that is intended to be used if you are using the external libraries feature. All your uploaded assets, thumbnails and transcoded videos are stored in `/photos`. Immich won't move your files from `/import` to `/photos`

**Reply by adminmat** [reply]
Ok i understand. I'd like to keep using the externa library method for now. Thanks for clarifying.
